src/ParkrunMap.UI/ParkrunMap.UI/ClientApp/src/components/NavMenu.js

```diff
@@ -37,20 +39,50 @@ export default class NavMenu extends React.Component {
             <NavbarToggler onClick={this.toggle} />
           </Container>
           <Collapse isOpen={this.state.isOpen} navbar>
-            <Nav className="ml-auto" navbar>
+            <Nav navbar>
+              <NavItem>
+                <NavLink
+                  tag={Link}
+                  active={this.context.router.route.location.pathname === '/'}
```

Comment:
Your router likely has a decorator function `withRouter`, so instead of fishing through the context directly (which is a big big code smell) you should be able to do:

```
@withRouter
export class default extends React.Component {
  render() {
    return (
      <NavLink active={this.props.match.location.pathname === '/'} />
    )
  }
}
```

or if not using decorators:
```
export default withRouter(NavMenu);
```